### PR TITLE
[visualization] configure visualization options

### PR DESCRIPTION
**Pull request by simonlingoogle**
This PR:
- Add the `cv` command for configuring visualization options
- Add `OTNS.config_visualization` to `pyOTNS`
- Turn off visualization broadcast messages  in the `farm` demo

### CODE REVIEW

File: dispatcher/visualization_options.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package dispatcher
```
Comment (wgtdkp) [inline dispatcher/visualization_options.go:1]: Add license header.

Comment (simonlingoogle) [reply dispatcher/visualization_options.go:27]: added

File: cli/README.md [hunk 2]
```diff
@@ -64,6 +65,40 @@ DispatchAllInRange                       0
...
+ack=off
+rtb=on
+ctb=on
+Done
+> cv bro on uni on ack on rtb on ctb on
+bro=on
+uni=on
+ack=on
+rtb=on
+ctb=on
```
Comment (wgtdkp) [inline cli/README.md:99]:
```suggestion
ctb=on
Done
```

File: pylibs/unittests/test_basic.py [hunk 3]
```diff
@@ -180,6 +180,38 @@ def assert_increasing(c0: Dict[str, int], c1: Dict[str, int]):
...
+        vopts = ns.config_visualization()
+        print('vopts', vopts)
+        for opt in ('broadcast_message', 'unicast_message', 'ack_message', 'router_table', 'child_table'):
+            self.assertTrue(opt in vopts)
+
+        vopts = ns.config_visualization(broadcast_message=False)
+        self.assertFalse(vopts['broadcast_message'])
+
+        vopts = ns.config_visualization(unicast_message=False)
+        self.assertFalse(vopts['broadcast_message'])
```
Comment (jwhui) [inline pylibs/unittests/test_basic.py:194]: Is testing `broadcast_message` after setting `unicast_message` intentional?

Comment (simonlingoogle) [reply pylibs/unittests/test_basic.py:194]: Actually yes. The `self.assertFalse(vopts['broadcast_message'])` statement is here to check if the configuration of `broadcast_message` is kept unchanged in `ns.config_visualization(unicast_message=False)`.

Comment (jwhui) [reply pylibs/unittests/test_basic.py:194]: Why not check other configurations?

Comment (simonlingoogle) [reply pylibs/unittests/test_basic.py:194]: yes, I will do that

Comment (simonlingoogle) [reply pylibs/unittests/test_basic.py]: revised the test

**Pull request merged**